### ISSUE TITLE
MemoryStore has no exit method and keeps the process up.

### ISSUE THREAD

**Issue by eavdmeer** [roccomuso/memorystore#24]
If all other resources in my program exit, which should stop the program, it keeps running because of MemoryStore:

```javascript
    store: this.config.sessionsPersistent ?
      new FileStore({ path: this.config.sessionDir, ttl: ttl }) :
      new MemoryStore({ checkPeriod: ttl * 1000 }),
```

With FileStore, the process exits. With MemoryStore, it hangs.

**Comment by roccomuso** [comment]
Did you call `stopInterval` on your memorystore instance?

https://github.com/roccomuso/memorystore/blob/master/lib/memorystore.js#L280

I might add an `unref` in a near future: https://httptoolkit.tech/blog/unblocking-node-with-unref/#timeoutunref
to avoid memorystore from keeping the process alive.

**Comment by eavdmeer** [comment]
That's actually a very good point. I did not stop the interval time. I'll have a look at it first thing in the morning! Thanks!

**Comment by eavdmeer** [comment]
I can confirm that this works as it should. After calling `stopInterval()`, the process exits correctly.

**Comment by copmerbenjamin** [comment]
Just for info:
Unref was implemented in https://github.com/roccomuso/memorystore/commit/e16e29b86c08588c0ecf1642e6264c40364a8ade and released in [1.6.7](https://github.com/roccomuso/memorystore/releases/tag/v1.6.4)
